src-java/testing/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/xresilience/RetriesSpec.groovy

```diff
@@ -385,7 +388,6 @@ class RetriesIsolatedSpec extends HealthCheckSpecification {
 
         cleanup:
         lockKeeper.cleanupTrafficShaperRules(swPair.src.regions)
-        islHelper.restoreIsl(islToBreak)
         database.resetCosts(topology.isls)
```

Comment:
it seems that we can omit it as block 
 cleanupManager.addAction(RESET_ISLS_COST,{database.resetCosts(topology.isls)}) was added above..